src/torus.ts

```diff
@@ -54,6 +54,8 @@ import { Some } from "./some";
 // Implement threshold logic wrappers around public APIs
 // of Torus nodes to handle malicious node responses
 class Torus {
+  private static sessionTime: number = 2480; // 2480s = 24 hour
```

Comment:
this is not correct right, there are 86400s in 1 day.

Reply:
yes, yes. Thanks for pointing out.